HDFDataset.py

```diff
@@ -965,9 +965,13 @@ def dump_from_dataset(self, dataset, epoch=1, start_seq=0, end_seq=float("inf"),
     print("Data keys:", data_keys, file=log.v3)
     if "orth" in data_keys:  # special workaround for now, not handled
       data_keys.remove("orth")
-    data_target_keys = [key for key in dataset.get_target_list() if key in data_keys]
+
+    if len(data_keys) > 1:
```

Comment:
Why is this needed?
Also, isn't this wrong?

Reply:
I don't see why this should be wrong.
If you only have one data_key, you want it to go into data_input_keys, isn't that right?

Reply:
Ah, sorry, you are right.